docs/spec.md

```diff
@@ -5010,12 +5010,14 @@ number of bits and produces a `result` tensor.
 #### Examples
 
 ```mlir
-// %lhs: [-1, -128, -36, 5, 3, 7]
-// %rhs: [1, 2, 3, 2, 1, 3]
-%result = "stablehlo.shift_right_logical"(%lhs, %rhs): (tensor<6xi8>, tensor<6xi8>) -> tensor<6xi8>
-// %result: [127, 32, 27, 1, 1, 0]
+// %lhs: [-1, 0, 1]
+// %rhs: [1, 2, 3]
+%result = "stablehlo.shift_right_logical"(%lhs, %rhs): (tensor<3xi64>, tensor<3xi64>) -> tensor<3xi64>
```

Comment:
The input `lhs = 1` and `rhs = 3` add some value for left shift. For right shift, it would be better to have an input like shiftArithRight(16, 3) = 1.